packages/roosterjs-content-model-markdown/lib/markdownToModel/convertMarkdownToContentModel.ts

```diff
@@ -0,0 +1,33 @@
+import { markdownProcessor } from './processor/markdownProcessor';
+import { mergeModel } from 'roosterjs-content-model-dom';
+import type { IEditor } from 'roosterjs-content-model-types';
+
+/**
+ * Convert the whole content to ContentModel with the given plain text
+ * @param editor The editor instance
+ * @param text The markdown text
+ * @param splitLinesPattern The pattern to split lines. Default is /\r\n|\r|\\n|\n/
+ * @returns The ContentModelDocument
+ */
+export function convertMarkdownToContentModel(
+    editor: IEditor,
+    text: string,
+    splitLinesPattern?: string
+) {
+    editor.formatContentModel(
```

Comment:
Suggest this API does not have dependency to editor. It should just take string as input, and ContentModelDocument as output

Reply:
This whole package should not have dependency to editor